### PR TITLE
User registration

### PR DESCRIPTION
**Pull request by sidharth-404**
ImageCanvas added

### CODE REVIEW

File: DigrammingTool-App/src/main/java/com/diagrammingtool/app/DigrammingToolAppApplication.java [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.web.bind.annotation.CrossOrigin;
 @SpringBootApplication
 public class DigrammingToolAppApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(DigrammingToolAppApplication.class, args);
+		System.out.println("App started !!!!");
```
Comment (proxolo1) [inline DigrammingTool-App/src/main/java/com/diagrammingtool/app/DigrammingToolAppApplication.java:12]: remove console

File: DigrammingTool-App/src/main/java/com/diagrammingtool/app/Dto/ResetPasswordRequest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package com.diagrammingtool.app.dto;
+
+
```
Comment (proxolo1) [inline DigrammingTool-App/src/main/java/com/diagrammingtool/app/Dto/ResetPasswordRequest.java:3]: Have you considered using lombok ?

File: DigrammingTool-App/src/main/java/com/diagrammingtool/app/Dto/ResetPasswordRequest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    public void setOtp(String otp) {
+        this.otp = otp;
+    }
+
+
+
+
+	public ResetPasswordRequest() {
+		super();
```
Comment (proxolo1) [inline DigrammingTool-App/src/main/java/com/diagrammingtool/app/Dto/ResetPasswordRequest.java:38]: why is it used here?

Comment (sidharth-404) [reply DigrammingTool-App/src/main/java/com/diagrammingtool/app/Dto/ResetPasswordRequest.java:38]: no

File: DigrammingTool-App/src/main/java/com/diagrammingtool/app/controller/CanvasImageController.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+@RestController
+@RequestMapping("/api/diagrammingtool")
+@CrossOrigin("*")
+public class CanvasImageController {
+	
+	@Autowired
+	private CanvasImageService service;
+	
+	@PostMapping("/saveDummyImage")
+	public ResponseEntity<CanvasImage> SaveImage(@RequestBody CanvasImage canvasImageDummy){
```
Comment (proxolo1) [inline DigrammingTool-App/src/main/java/com/diagrammingtool/app/controller/CanvasImageController.java:29]: follow Java naming conventions.

File: DigrammingTool-App/src/main/java/com/diagrammingtool/app/controller/UserLoginController.java [hunk 5]
```diff
@@ -1,34 +1,94 @@
...
+    @Autowired
     private UserLoginServiceImpl userLoginService;
-
-    @PostMapping("/login")
+	
+	@Autowired
+	private JwtUtil jwtUtil;
+	
+	 @PostMapping("/login")
     public ResponseEntity<?> UserRegistration(@RequestBody UserLogin userLogin) {
```
Comment (proxolo1) [inline DigrammingTool-App/src/main/java/com/diagrammingtool/app/controller/UserLoginController.java:36]: method name should start in lowercase.

Comment (proxolo1) [inline DigrammingTool-App/src/main/java/com/diagrammingtool/app/controller/UserLoginController.java:36]: consider small caps

**Review by proxolo1 — APPROVED** [review]

**Pull request merged**